node_modules/oae-principals/lib/internal/dao.js

```diff
@@ -82,8 +83,14 @@ var createUser = module.exports.createUser = function(user, callback) {
  */
 var createGroup = module.exports.createGroup = function(groupId, tenantAlias, displayName, description, visibility, joinable, createdBy, callback) {
     var lastModified = Date.now();
+    var created = new Date(lastModified);
+
+    lastModified = lastModified.toString();
+    created = created.toUTCString();
```

Comment:
Not ideal, but may actually have ended up helping us with this particular PR, since the cassandra timestamps were causing problems. Will be difficult to fix in the future though since it'd require migrating all the tables using `text` at the moment.